spec/spec_helper.rb

```diff
@@ -59,3 +59,11 @@
   # https://relishapp.com/rspec/rspec-rails/v/3-0/docs
   config.infer_spec_type_from_file_location!
 end
+
+def login_user(user = nil)
```

Comment:
This could be refactored to something like:

```ruby
def set_current_user(user = nil)
  user ||= Fabricate(:user)
  session[:user_id] = user.id
end
```

and you should put it to some other file, in `spec/support` So it would get autoloaded here.